src/site/content/en/blog/badging-api/index.md

```diff
@@ -126,108 +98,58 @@ and users must add it to their home screens.
 make calls to the Badging API dependent on the install state. Just call the
 API when it exists, as other browsers may show the badge in other places.
 If it works, it works. If not, it simply doesn't.
 
-## Supporting both the first and second origin trial {: #backwards-compat }
-
-In Chrome 79, a [second origin trial]({{origin_trial.url}}) was started. Based
-on developer feedback received during the first origin trial, the API has
-changed, and is not backwards compatible with the first origin trial.
-
-If you have an existing implementation, and want to provide backwards
-compatibility for your users, you can wrap the calls in the following
-wrapper function. Once all of your users have migrated to Chrome 79 or later,
-you can remove this wrapper function from your code.
-
-```js
-function setBadge(...args) {
-  if (navigator.setExperimentalAppBadge) {
-    navigator.setExperimentalAppBadge(...args);
-  } else if (window.ExperimentalBadge) {
-    window.ExperimentalBadge.set(...args);
-  }
-}
-
-function clearBadge() {
-  if (navigator.clearExperimentalAppBadge) {
-    navigator.clearExperimentalAppBadge();
-  } else if (window.ExperimentalBadge) {
-    window.ExperimentalBadge.clear();
-  }
-}
-```
-
-{% Aside %}
-Be sure to include both the
-[first (Chrome 78 and earlier)](https://developers.chrome.com/origintrials/#/view_trial/1711367858400788481),
-and
-[second (Chrome 79 and later)]({{origin_trial.url}})
-origin trial tokens in your page.
-{% endAside %}
-
-## Feedback {: #feedback }
-
-The Chrome team wants to hear about your experiences with the Badging API.
-
-### Tell us about the API design
-
-Is there something about the API that doesn't work like you expected? Or are
-there missing methods or properties that you need to implement your idea?
-Have a question or comment on the security model?
-
-* File a spec issue on the [Badging API GitHub repo][issues], or add your
-  thoughts to an existing issue.
-
 ### Report a problem with the implementation
 
 Did you find a bug with Chrome's implementation? Or is the implementation
 different from the spec?
 
-* File a bug at <https://new.crbug.com>. Be sure to include as much detail
-  as you can, simple instructions for reproducing, and set
-  Components to `UI>Browser>WebAppInstalls`. [Glitch](https://glitch.com) works great for
-  sharing quick and easy repros.
+* File a bug at <https://new.crbug.com>. Be sure to include as much detail as
+  you can, simple instructions for reproducing, and set **Components** to
+  `UI>Browser>WebAppInstalls`. [Glitch](https://glitch.com) works great for
+  sharing quick and easy reproductions.
 
 ### Show support for the API
 
-Planning to use Badging API on your site? Your public support helps the Chrome
-team to prioritize features, and shows other browser vendors how critical it
-is to support them.
+Planning to use the Badging API on your site? Your public support helps the
```

Comment:
Policy thing: do we want to keep this section now that the API is launching? Is this a thing we ask people to do for all APIs, or just ones in a trial?

Reply:
I think it's good to still ask for support, it helps other browser vendors see people are using it, and it also potentially points us to new sites that are using it that we can point people at.